spec/requests/admin/overview_spec.rb

```diff
@@ -8,24 +8,6 @@
     allow(FeatureFlag).to receive(:enabled?).and_call_original
   end
 
-  describe "profile admin feature flag" do
```

Comment:
The overview page has changed and profile fields and DUS scripts are no longer shown on the overview page